scripts/run_tsfresh.py

```diff
@@ -0,0 +1,54 @@
+'''
+Run the script with:
+```
+python run_tsfresh.py path_to_your_csv.csv
+```
+e.g.:
+```
+python run_tsfresh.py data.txt
+```
+
+A corresponding csv containing time series features will be 
+saved as features_path_to_your_csv.csv
+'''
+
+import sys
+import numpy as np
+import pandas as pd
+from tsfresh import extract_features, extract_relevant_features, select_features
+
+
+def _preprocess(df):
+	'''
+	given a dataframe where records are stored row-wise, rearrange it 
```

Comment:
You could specify the format a little bit more. I think you wrote this method for dfs where every row has one kind of time series